docs/Project and Community/FAQ.rst

```diff
@@ -53,23 +51,23 @@ requires.
 Installation
 ------------
 
-ZFS on Linux is available for all major Linux distributions. Refer to
-the :doc:`getting started <../Getting Started/index>` section of the wiki for links to installations
-instructions for many popular distributions. If your distribution isn't
-listed you can always build ZFS on Linux from the latest official
-`tarball <https://github.com/zfsonlinux/zfs/releases>`__.
+OpenZFS is available for all major Linux distributions and FreeBSD. Refer to
```

Comment:
Maybe use "FreeBSD and all major Linux distributions"? My theory here would be that the list of co-equal platforms should be alphabetical (to avoid giving the appearance that Linux is primary). If that seems reasonable, then I'd do the same below for the list of support architectures: aarch64, arm, ppc, ppc64, x86, x86_64

Reply:
Good idea, changed.